app/models/__init__.py

```diff
@@ -157,6 +157,7 @@
 QEMU_KEY = "qemu"
 RESULT_KEY = "result"
 RETRIES_KEY = "retries"
+ROOTFS_KEY = "rootfs"
```

Comment:
Please factor out the changes with the "rootfs" key, as doing that will require other parts of the code to be updated (emails make use of it).

Reply:
This now applies to #275